helm/net-exporter-chart/templates/rbac.yaml

```diff
@@ -0,0 +1,34 @@
+kind: RoleBinding
```

Comment:
Nitpick: I'd prefer to have RoleBinding specified below the role. That read better.

Reply:
Why not. Done!